### PR TITLE
Add haskell-language-server-bin to Arch Linux section

### PR DESCRIPTION
**Pull request by marcin-rzeznicki**
See #1634

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,8 @@ To install, download the `haskell-language-server-wrapper` executable for your p
 
 ### Arch Linux
 
+The preferred method of installation for development purposes is to use the [haskell-language-server-bin](https://aur.archlinux.org/packages/haskell-language-server-bin) package from AUR. This package contains statically linked binaries for each supported GHC version and `haskell-language-server-wrapper` for dynamic GHC version selection. It is updated regularly, requires no additional dependencies, and is independent of other haskell packages you may have on your system, including GHC. Its size is relatively large (approx. 900 MB), but if this is a problem for you, during installation you can disable the GHC versions you will not be using by editing the PKGBUILD file.
+
```
Comment (Ailrun) [inline README.md:144]: I think it's maybe better fo update the following paragraph so that they flow better.

**Review by Ailrun — COMMENTED** [review]
For following paragraphs

File: README.md (hunk 1 above)
Comment (marcin-rzeznicki) [reply README.md:144]: I have to admit that I like how they flow and it seems logical to me: The preferred method (...) is (...) . If you're using (...) then (...). But I'm open to suggestions

Comment (Ailrun) [reply README.md:144]: I mean we may put some connetion words like
```diff
- If you are using ArchLinux with **dynamically linked** ... you can ...
+ Alternatively if you are using **dynamically linked** ... you can also ...
```
and removing "ArchLinux" as this paragraph is already quite far from the section title, it feels a bit awkward to mention "ArchLinux" again...
Though I'm not strongly inclined to this.

Comment (marcin-rzeznicki) [reply README.md:144]: Ah, good idea

Comment (marcin-rzeznicki) [reply README.md:144]: Fixed in 7d7d83e

File: README.md [hunk 2]
```diff
@@ -140,7 +140,9 @@ To install, download the `haskell-language-server-wrapper` executable for your p
 
 ### Arch Linux
 
-If you are using Arch Linux with **dynamically linked** Haskell packages from `pacman`,
+The preferred method of installation for development purposes is to use the [haskell-language-server-bin](https://aur.archlinux.org/packages/haskell-language-server-bin) package from AUR. This package contains statically linked binaries for each supported GHC version and `haskell-language-server-wrapper` for dynamic GHC version selection. It is updated regularly, requires no additional dependencies, and is independent of other haskell packages you may have on your system, including GHC. Its size is relatively large (approx. 900 MB), but if this is a problem for you, during installation you can disable the GHC versions you will not be using by editing the PKGBUILD file.
```
Comment (berberman) [inline README.md:143]: Please break this single-line paragraph into several lines, now it's too long!

Comment (berberman) [inline README.md:143]: Avoid using words like "dynamic GHC version", which may confuse with dynamically linked GHC

Comment (marcin-rzeznicki) [reply README.md]: Sorry for the delay. Fixed in 203da29

Comment (marcin-rzeznicki) [reply README.md]: Replaced with `automatic` in 203da29

**Review by berberman — APPROVED** [review]
LGTM, thanks!

**Pull request merged**